.github/workflows/build.yml

```
@@ -51,6 +51,57 @@
+          # and this is necessary when a workflow is triggered by something other than pushing a tag
+          fetch-depth: 0
+
+      - name: Get previous tag
+        if: steps.skip_check.outputs.should_skip != 'true'
+        id: prev_tag
+        run: |
+          echo "::set-output name=prevtag::$(git describe --tags --abbrev=0 HEAD^1)"
+
+      - name: DEBUG Show GITHUB_REF
+        shell: pwsh
+        run: echo "GITHUB_REF: $env:GITHUB_REF"
```

actionlint

could not parse as YAML: yaml: line 66: mapping values are not allowed in this context

(syntax-check)